packages/cashier/src/components/alert-banner/__tests__/alert-banner.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import AlertBanner from '../alert-banner';
+
+describe('<AlertBanner />', () => {
+    it('should render the icon and message props passed for the banner', () => {
+        const props: React.ComponentProps<typeof AlertBanner> = {
+            icon: 'IcAlertWarningDark',
+            message: 'Alert banner test messsage.',
```

Comment:
```suggestion
            message: 'Alert banner test message.',
```
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker